echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/bearychat/BearychatNotificationService.groovy

```diff
@@ -40,8 +39,8 @@ class BearychatNotificationService implements NotificationService {
   String token
 
   @Override
-  boolean supportsType(Notification.Type type) {
-    return type == TYPE
+  boolean supportsType(String type) {
+    return "BEARYCHAT".equals(type.toUpperCase())
```

Comment:
Nit: I wonder if we should have the type be a constant member field on the class, like `private static final TYPE = "BEARYCHAT"`.  It may make this code easier to read.

Reply:
The only reason that I didn't was because it is only used once. If there were multiple usages I'd definitely do something like that. In the future, I'd like to do something that combines supportsType and getNotificationType.